src/main/java/seedu/address/model/person/Person.java

```diff
@@ -163,11 +173,25 @@ public String toString() {
         return builder.toString();
     }
 
+    /**
+     * Returns true if a person is a doctor.
+     * Returns false otherwise.
+     */
     public boolean isDoctor() {
+        if (role.toString().equals("Doctor")) {
+            return true;
+        }
         return false;
     }
 
+    /**
+     * Returns true if a person is a patient.
+     * Returns false otherwise.
+     */
     public boolean isPatient() {
```

Comment:
Since Patient and Doctor extends Person, I was wondering if using `person instanceof Patient` is sufficient to determine whether Person is a Patient or a Doctor?